src/ape_accounts/accounts.py

```diff
@@ -257,3 +259,39 @@ def __decrypt_keyfile(self, passphrase: str) -> HexBytes:
             return EthAccount.decrypt(self.keyfile, passphrase)
         except ValueError as err:
             raise InvalidPasswordError() from err
+
+
+def generate_account(
+    alias: str, passphrase: str, hd_path: str = ETHEREUM_DEFAULT_PATH, word_count: int = 12
+) -> Tuple[AccountAPI, str]:
+    """
+    Generate a new account.
+
+    Args:
+        extra_entropy (str): Extra entropy to use when generating the account.
+
+    Returns:
+        Tuple of AccountAPI and mnemonic for the generated account.
+    """
+    EthAccount.enable_unaudited_hdwallet_features()
+    accounts = AccountManager()
+
+    if alias in accounts.aliases:
+        raise AliasAlreadyInUseError(alias)
+    elif not isinstance(alias, str):
+        raise AccountsError(f"Alias must be a str, not '{type(alias)}'.")
+    elif is_hex(alias) and len(alias) >= 42:
+        # Prevents private keys from accidentally being stored in plaintext
+        # Ref: https://github.com/ApeWorX/ape/issues/1525
+        raise AccountsError("Longer aliases cannot be hex strings.")
+
+    if not passphrase or not isinstance(passphrase, str):
+        raise AccountsError("Account file encryption passphrase must be provided.")
```

Comment:
This would change to `click.prompt` w/ `hide_input=True`

Reply:
Not sure what you mean

Reply:
Related to above